manifests/pod.yaml

```diff
@@ -54,6 +54,9 @@ spec:
      - name: fedora-messaging-coreos-key
        mountPath: /run/kubernetes/secrets/fedora-messaging-coreos-key
        readOnly: true
+     - name: oscontainer-registry
```

Comment:
The pod shouldn't need the secret mounted in since we're using `withCredentials`.